python/pyspark/ml/clustering.py

```diff
@@ -141,7 +132,7 @@ def summary(self):
         training set. An exception is thrown if no summary exists.
```

Comment:
I think you can then just eliminate these implementations, as they'll do just the same as the superclass. We lose the specialized doc comment, but, that's part of unifying these I think.

Reply:
@srowen Thank you very much for your review. I think I still need  the ```summary``` implementation here, because I need to cast the ```super.summary``` to ```GaussianMixtureSummary```. If I remove the implementation, the unit test will fail. 
I checked the scala code, it also has ```summary``` implementation in the specific subclass. Seems removing the implementation will have error too.